receiver/splunkenterprisereceiver/factory.go

```diff
@@ -61,14 +61,13 @@ func createMetricsReceiver(
 ) (receiver.Metrics, error) {
 	cfg := baseCfg.(*Config)
 	splunkScraper := newSplunkMetricsScraper(params, cfg)
-
 	s, err := scraper.NewMetrics(
 		splunkScraper.scrape,
 		scraper.WithStart(splunkScraper.start))
 	if err != nil {
+		splunkScraper.settings.Logger.Error(err.Error())
```

Comment:
Is this log line of value here considering this is on component create, this should already give us the issue observed?

Reply:
You're probably right, I can remove it.